spanner/spec/spanner_samples_spec.rb

```diff
@@ -1338,4 +1402,175 @@ def capture
     expect(captured_output).to include "11.1212"
     expect(captured_output).to include "104.4123101"
   end
+
+  example "create backup" do
+    expect(@instance.backup(@backup_id)).to be nil
+    database = create_database_with_data
+
+    capture do
+      create_backup project_id:  @project_id,
+                    instance_id: @instance.instance_id,
+                    database_id: database.database_id,
+                    backup_id:   @backup_id,
+                    expire_time: @expire_time
+    end
+
+    expect(captured_output).to include(
+      "Backup operation in progress"
+    )
+    expect(captured_output).to include(
+      "Backup #{@backup_id} of size 607 bytes was created at"
+    )
+
+    @test_backup = @instance.backup @backup_id
+    expect(@test_backup).not_to be nil
+  end
+
+  example "restore backup" do
+    expect(@instance.backup(@backup_id)).to be nil
+    backup = create_backup_with_data
```

Comment:
Unfortunately, we shouldn't create a backup for each test because it will take forever for these tests to run. Is it possible to just create a backup once and reuse it?

The way the other client library sets up the tests is to run them in some order such that each previous test creates what is required in later tests. E.g.

1. create_backup
2. cancel_backup
3. update_backup
4. list_backups
5. list_backup_operations
6. restore_backup
7. list_database_operations
8. delete_backup